js/index.js

```diff
@@ -39,4 +39,126 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+// Button
+var btn = document.querySelector('.cta .cta-text button');
+btn.textContent = "Get Started";
+
+// Header
+var ctaText = document.getElementsByTagName('h1');
+ctaText[0].textContent = "DOM Is Awesome";
+console.log(ctaText)
+
+// Images
+var middleImg = document.querySelector('#middle-img');
+middleImg.src = "img/mid-page-accent.jpg";
+middleImg.alt = "Image of code snippets across the screen";
+
+var logoImg = document.querySelector('#cta-img');
+logoImg.src = "img/header-img.png";
+logoImg.alt = "Great Idea! Company logo.";
+
+// Navigation Links
+const nav = document.getElementsByTagName('a');
+
+var services = nav[0];
```

Comment:
This is a great place where we could apply the D.R.Y. principle. We could maybe use `forEach` here with some interpolation:
```javascript
navs.forEach((anchor, i) => anchor.textContent = siteContent['nav'][`nav-item-${i + 1}`]);
```